src/main/java/mate/academy/springboot/swagger/util/Queries.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.swagger.util;
+
+public class Queries {
+    public static final String PRODUCT_QUERY_VALUE = "SELECT * FROM products "
+            + "AS p WHERE p.price BETWEEN ?1 AND ?2";
+    public static final String PRODUCT_QUERY_COUNT = "SELECT count(*) FROM products "
+            + "AS p WHERE p.price BETWEEN ?1 AND ?2";
+    public static final String GET_MAX_PRICE_QUERY = "SELECT max(products.price) "
+            + "FROM products";
+    public static final String GET_MIN_PRICE_QUERY = "SELECT min(products.price) "
+            + "FROM products";
+
+}
```

Comment:
it's looking good for this project may be but in the real one you won't do like this because there will be a lot of such queries maybe even hundreds, so it'll be looking eventually like a mess, so please remove this class and move queries to the repository where they belong